packages/@expo/config-types/build/ExpoConfig.d.ts

```diff
@@ -294,6 +294,22 @@ export interface Splash {
     image?: string;
     [k: string]: any;
 }
+
```

Comment:
Unfortunately this is not the correct way to change the types in the Expo config:
- The types are actually derived from the internal schema used in our services, so the next schema update would wipe out this change.
- If you did change them here manually, they would have to be changed in the source (`packages/@expo/config-types/src/ExpoConfig.ts`) and then the module must be rebuilt.

Reply:
I understand, no worries. I figured that is the case based on @amandeepmittal 's comments. In the very least, I hope this PR helps steer the Expo team in a good direction regarding implementing support for this. Thanks for taking a look at this nevertheless! Feel free to close the PR if needed.

Reply:
We appreciate the heads up about this feature, and thanks for the PR! 👍